src/__tests__/CommunicationErrors.tests.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Italia PagoPA Proxy
```

Comment:
as these are integration tests and not unit test, please rename the folder in `__integration__`